plugins/crossplane-backend/README.md

```
@@ -0,0 +1,172 @@
+  allowUnauthenticated: true  # ⚠️ ONLY for development!
+```
+
+**Security Notes:**
+- **Default:** `allowUnauthenticated: false` - requires user authentication via cookie/token
+- **Production:** Never set `allowUnauthenticated: true` in production environments
+- **Development:** Only use `allowUnauthenticated: true` for local testing
+- When authenticated, users access Crossplane resources based on their permissions
+
+### How Authentication Works
+
+**1. Browser-based (Recommended for Development)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)